class DataNotTrainableError(Exception):
    def __init__(self, fname):
        pass
